src/App.tsx

```diff
@@ -34,6 +35,20 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort((firstGood, secondGood) => (
+      firstGood.localeCompare(secondGood)
+    ));
+  } else if (sortType === SortType.LENGTH) {
+    visibleGoods.sort((firstGood, secondGood) => (
+      firstGood.length - secondGood.length
+    ));
+  }
+
+  if (isReversed === true) {
+    visibleGoods.reverse();
+  }
```

Comment:
call method sort 1 time and use  `switch` isnside of sort

Reply:
Will do, thanks)